charters/web5-working-group.md

```diff
@@ -0,0 +1,133 @@
+
+* Conformance infrastructure such as test vectors, test suites to encourage compatible implementations across programming languages.
+
+* Enable the development of Decentralized Applications (dApps) and Protocols using technologies such as Decentralized Identifiers (DIDs), Verifiable Credentials (VCs), and Decentralized Web Nodes (DWNs).
+
+## Deliverables
+
+### 1. Web5 Specification
+
+**Description**: Develop and maintain the [Web5 Specification](https://github.com/TBD54566975/web5-spec/), that clearly specify:
+
+**DRI**: [Gabe Cohen](https://github.com/decentralgabe)
+
+**Scope of Work**:
+
+* Specification of the digital identity standards we will implement (e.g. specific DID methods, verifiable credential data models, credential issuance protocols, storage, etc.) along with specific implementation details (e.g. required features, minimal subsets, etc.).
+
+* An API design (OpenAPI specification) used to interact with the digital identity standards we implement, able to be uptaken across Web5 SDKs. Note: this API should be directly influenced by the API design work item from the Rust Working Group.
+
+* Test vectors from which conformance to each feature in the specification can be achieved.
+
+### 2. Web5 SDKs
+
+**Description**: Develop and maintain Web5 SDKs which shall conform to the Web5 Specification.
+
+**DRI**: [Frank Hinek](https://github.com/frankhinek)
+
+**Scope of Work**:
+
+* SDKs in [Typescript](https://github.com/TBD54566975/web5-js), [Kotlin](https://github.com/TBD54566975/web5-kt), [Swift](https://github.com/TBD54566975/web5-swift), [Dart](https://github.com/TBD54566975/web5-dart), [Go](https://github.com/TBD54566975/web5-go), and [Rust](https://github.com/TBD54566975/web5-rs).
+
+* SDKs shall demonstrate conformance to the test vectors defined by the Web5 specification.
+
+* SDKs shall be well-documented according to each language's conventions, as well as being surfaced through the [developer site](https://developer.tbd.website/docs/).
+
+* SDKs shall follow the guidance of the Rust Working Group in adopting a Rust core to limit developer overhead. 
+
+* The JS SDKs shall follow the guidance of the DWeb Working Group in maintaing compatability with Decentralized Web Nodes (DWNs).
+
+### 3. DID DHT
+
+**Description**: Develop and maintain the DHT Decentralized Identifier method (did:dht).
+
+**DRI**: [Gabe Cohen](https://github.com/decentralgabe)
+
+**Scope of Work**:
+
+* Develop and maintain the [DID DHT Method Specification](https://did-dht.com/) and [DID DHT Method Registry](https://did-dht.com/registry).
+
+* Develop and maintain an open source implementation of a [DID DHT Gateway Server](https://did-dht.com/#gateways).
+
+* Develop and maintain a conformance testing suite for DID DHT clients, including test vectors and implementation reports.
+
+* Ensure conformant implementation of the DID DHT method across **Web5 SDK** implementations.
+
+## Success Criteria
```

Comment:
I'm very curious about this being an actual specification, because absent a section pointing at the DWN spec, it would just be a relabling of the DID and VC specs. Are we going to mention and ensure it includes the datastore aspects of the stack that make Web5, Web5?

Reply:
that's not currently in scope. if you think that should be in scope please make a concrete suggestion.

tbDEX as it stands today has no need for this